### PR TITLE
Responsive map

### PR DESCRIPTION
**Pull request by aabdaab**
* Add responsive styles for the Map
* Fix map controls styles
* Fix map dropdowns styles

image

Screenshot 2019-12-16 at 11 48 47

### CODE REVIEW

File: app/javascript/components/world-map/MapLegend.js [hunk 1]
```diff
@@ -51,8 +51,8 @@ function MapLegend({ content, context, scales }) {
               <div>
                 <div className="rectangle" style={{backgroundColor: color}} />&nbsp;
 
-              {(i === 0) && `<${displayContextValue(color, 1)}`}
-              {(i === colors.length - 1) && `>${displayContextValue(color, 0)}`}
+                {(i === 0) && `<${displayContextValue(color, 1)}`}
```
Comment (tsubik) [inline app/javascript/components/world-map/MapLegend.js]: emacs keeps messing this up lol

File: app/javascript/components/world-map/WorldMap.js [hunk 2]
```diff
@@ -58,6 +59,42 @@ function WorldMap({ zoomToGeographyIso }) {
   // react-select
   const setContentId = selectedOption => dispatch({ type: 'setContentId', payload: selectedOption.value });
   const setContextId = selectedOption => dispatch({ type: 'setContextId', payload: selectedOption.value });
+  const customStyles = {
```
Comment (tsubik) [inline app/javascript/components/world-map/WorldMap.js]: I'm totally not a fan of CSS in javascript, but as react-select went this way :man_shrugging:

Comment (tsubik) [reply app/javascript/components/world-map/WorldMap.js]: That's why I haven't styled it before, procrastinating things I don't like, but tell no-one :P

**Comment by aabdaab** [comment]
As a bonus, I added here the fix for broken assets in the search results page. I'm not happy about that solution because we might end up adding more assets to the public folder if we want to use them a similar way but have failed to fix that with rails webpacker :/

**Pull request merged**